src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -72,10 +73,10 @@
      * 3. Returning "true" will add the provided task to the list of updated tasks, if not there
      *    already.
      *
-     * @param other The other task
+     * @param that The other task
      * @return True if it can be, false if not
      */
-    boolean canRunConcurrentWith(AITask<O> other);
+    boolean canRunConcurrentWith(AITask<?> that);
```

Comment:
> Strictly speaking, the compile errors are more of an implementation problem (as you said above) and not my problem as the API (not to be rude). This means `? super O` IS "better" than just `?` as you have an upper-bound. With just ?, any AITask with any type can be passed in.

Question: What happens when you try to pass, say, `AITask<Monster>` to `AITask<Living>.canRunConcurrentWith()`? `Monster` is not a superinterface of `Living`.